src/debugger/launcher.ts

```diff
@@ -49,14 +49,17 @@ export class Launcher {
                         // We've seen that if we don't prewarm the bundle cache, the app fails on the first attempt to connect to the debugger logic
                         // and the user needs to Reload JS manually. We prewarm it to prevent that issue
                         .then(() => {
+                            Log.logMessage("Prewarming bundle cache. This may take a while...");
```

Comment:
I'd move this message under the generator.step("prewarmBundleCache") line, so it'll make more in the weird case we get a crash in that line.